docs/source/interfaces/drill.rst

```diff
@@ -0,0 +1,140 @@
+DrILL interface
+===============
+
+DrILL stands for Data Reduction at ILL. It provides an easy to use interface for
+the reduction of data measured on different ILL instruments. Here are the
+currently supported instruments:
+
+    * D11: SANS
+    * D16: SANS
+    * D17: Reflectometry
+    * D22: SANS
+    * D33: SANS
+    * FIGARO: Reflectometry
+
+The interface is accessible through the Workbench menu bar: *Interfaces* ->
```

Comment:
Mention that the facility has to be set to ILL before.

Reply:
Done.